modules/mobile/app/serializers/mobile/v0/user_serializer.rb

```diff
@@ -42,7 +42,7 @@ class UserSerializer
         appeals: :appeals,
         appointments: :vaos,
         claims: :evss,
-        directDepositBenefits: %i[evss ppiu],
+        directDepositBenefitsView: %i[evss ppiu],
```

Comment:
Have you checked with @narin and the FE crew that it's ok to change an authorized service name? I think adding one is fine (following the [API evolution](https://apisyouwonthate.com/blog/api-evolution-for-rest-http-apis) strategy) but otherwise we might have to version the endpoint.

Reply:
That makes sense. I'll check with the FE

Reply:
FE let me know they use this so I reverted it to the original. directDepositBenefits in authorized_services indicates view access only